### ISSUE TITLE
Pod support

### ISSUE THREAD

**Issue by GabrielCW** [jamesarosen/ember-i18n#346]
Hi there, I was wondering if there are any plans to support pods to load translations? It seems Ember is gonna support pods more and more, so it would be appreciated to have a way to dispatch translations in pods.

Is there a way to do this already? What would be missing to make it work?
The subject was touched in #255 but I haven't seen it discussed elsewhere.

Edit: there seems to be an [addon](https://www.npmjs.com/package/ember-cli-locales-pods) doing that, but wouldn't it be useful to port this into ember-i18n?

**Comment by jamesarosen** [comment]
I have not given this any thought yet. Do you want to put together an RFC? Or maybe just list some requirements and try to raise some of the relevant questions? For example:

 * How does Ember-I18n find all of the translations modules?
 * If two pods have conflicting translations, which wins?

**Comment by null-null-null** [comment]
You may have already seen [this video](https://www.youtube.com/watch?v=1wc3a_ttTy0&feature=youtu.be&t=28m18s), but pod structure will be changing considerably in the (ambiguously) near future. Might be worth holding off until the dust has settled there.

**Comment by jamesarosen** [comment]
Thanks for the tip! It will be several weeks before I pick up more ember-i18n work. I'll be sure to keep this in mind when I do 😀

**Comment by NullVoxPopuli** [comment]
I'm testing out module unification atm,

had to change my resolver.js to look like this:
```
import Resolver from 'ember-resolver/resolvers/fallback';
// import Resolver from 'ember-resolver/resolvers/glimmer-wrapper';

import { merge } from '@ember/polyfills';


import buildResolverConfig from 'ember-resolver/ember-config';
import config from '../config/environment';

let moduleConfig = buildResolverConfig(config.modulePrefix);
/*
 * If your application has custom types and collections, modify moduleConfig here
 * to add support for them.
 */

 merge(moduleConfig.types, {
  config: { definitiveCollection: 'main' },
  locale: { definitiveCollection: 'main'},
  'ember-i18n@config': { definitiveCollection: 'main' },
  'util': { definitiveCollection: 'utils' },
});

moduleConfig.collections.main.types.push('config');

export default Resolver.extend({
  config: moduleConfig
});
```

can't find my translations yet (using typescript). more research ahead!

**Comment by NullVoxPopuli** [comment]
ok, I have translations kinda working... but it's a hack.

ember-i18n doesn't support the `src` (as opposed to app paths) paths that module unification is uses.

to remedy this for the short term, I did:
```
cd <project>
mkdir app/locales/en-us
cd app/locales/en-us
ln -s ../../../src/locales/en-us/translations.js transaltions.js
```

the location of my actual file is at `<project>/src/locales/en-us/translations.js` (I also changed the extension to js, because ts wasn't being picked up by ember-i18n (I probably just needed to tweak my tsconfig... but oh well - ember-i18n doesn't have types as for as I'm aware of anyway))

![image](https://user-images.githubusercontent.com/199018/39725360-736bf102-5219-11e8-8102-8f38faafb9f2.png)

I have `app` `.gitignore`d because with module unification, there should be nothing in the app directory (but it keeps getting added because generators aren't quite up to date yet.

here is my repo if you want to go poke around: https://github.com/NullVoxPopuli/emberclear
